src/SimpleThings/EntityAudit/AuditReader.php

```diff
@@ -748,12 +748,24 @@ public function getEntityValues($className, $entity)
     {
         $metadata = $this->em->getClassMetadata($className);
         $fields = $metadata->getFieldNames();
+        $associationsFields = $metadata->getAssociationNames();
 
         $return = array();
         foreach ($fields AS $fieldName) {
             $return[$fieldName] = $metadata->getFieldValue($entity, $fieldName);
         }
 
+        foreach($associationsFields as $key => $association) {
+            $associationValues = $entity->{'get' . ucfirst($association)}();
```

Comment:
No, `property access` use getter methods to access the values. Doctrine doesn't use class methods. It uses class/property reflections to access the properties directly.